test/e2e/fleet_guard_rail_test.go

```diff
@@ -513,3 +515,87 @@ var _ = Describe("fleet guard rail for UPDATE work operations, in fleet prefixed
 		}, eventuallyDuration, eventuallyInterval).Should(Succeed())
 	})
 })
+
+var _ = Describe("fleet guard rail networking E2Es", Serial, Ordered, func() {
+	BeforeAll(func() {
```

Comment:
Hi Arvind! Also a nit, but this back-and-forth installation/uninstallation of CRDs is really not necessary.